net/can/can_setsockopt.c

```diff
@@ -76,7 +76,7 @@ int can_setsockopt(FAR struct socket *psock, int option,
   int ret = OK;
   int count = 0;
 
-  DEBUGASSERT(psock != NULL && value != NULL && psock->s_conn != NULL);
+  DEBUGASSERT(psock != NULL && psock->s_conn != NULL);
```

Comment:
why not change to:
DEBUGASSERT(psock != NULL && psock->s_conn != NULL);
DEBUGASSERT(value_len == 0 || value != NULL);

Reply:
Right, it's better. Fixed